pkgs/applications/misc/openrgb/default.nix

```diff
@@ -0,0 +1,55 @@
+{ stdenv
+, qmake
+, fetchFromGitHub
+, fetchFromGitLab
+, libusb1 
+, hidapi
+, qt5
+, wrapQtAppsHook
+, makeWrapper
+, hicolor-icon-theme
+, openrazer-daemon
+}:
+stdenv.mkDerivation rec {
+  pname = "openrgb";
+  version = "unstable-${date}";
+
+  date = "2020-03-03";
+
+  src = fetchFromGitHub {
+    owner = "CalcProgrammer1";
+    repo = "OpenRGB";
+    rev = "d7298cafd02232a4da3843a5e6c810cb2ccd99f4";
+    sha256 = "0gjj8z3574dxx41d38khhq91c6g2z5jiws5k9hg0lryks97341ak";
+    fetchSubmodules = true;
+  };
+
+  buildInputs = [ libusb1 hidapi wrapQtAppsHook ];
+
+  propagatedBuildInputs = [ hicolor-icon-theme openrazer-daemon ];
```

Comment:
We probably don't actually need `hicolor-icon theme`, this was probably added when I was trying to figure out why the application was segfaulting